### ISSUE TITLE
Support for Chinese Documentation

### ISSUE THREAD

**Issue by Waterkin** [apache/dolphinscheduler-sdk-python#136]
Support for Chinese Documentation for DolphinScheduler better spreading in Chinese developer community.

**Comment by zhongjiajie** [comment]
Maintaining two versions of the doc is not a good idea, maybe we should find some way to maintain only English version doc and  auto translate doc to Chinese
 have a related issue for that, you can see https://github.com/apache/dolphinscheduler/issues/8233

**Comment by Waterkin** [comment]
@zhongjiajie Agreed. I am willing to add a translate button to pydolphinscheudle doc. However, I am wondering how to build the doc locally. It would be better to have a README.md about how to build the doc under `/docs`.

**Comment by zhongjiajie** [comment]
We have it in contribution doc https://github.com/apache/dolphinscheduler-sdk-python/blob/main/CONTRIBUTING.md#build-document
